text/0000-guides-restructure.md

```diff
@@ -0,0 +1,397 @@
+
+## Drawbacks
+
+This refactor is biased towards new user experience, so existing Ember users could experience the most drawbacks.
+
+1. They will need to discover where old content lives. This is mitigated by the site search, which is now stabilized
+2. Experienced developers who haven’t looked at the Guides for a long time will all reference it during the Octane upgrade, and may be surprised to find a new layout. It’s another “new thing to learn.” This is mitigated by consolidating most of the new things to learn into the Components section
+3. There will likely be some wrinkles to iron out with regards to content that should have been refactored during rearranging, but was overlooked. We are confident that the community will help identify these issues.
+
+## Alternatives
+
+The main architectural pattern choices were as follows:
+
+1. On one extreme, make every section standalone
+2. The middle ground, establish a “Common Core” to be read sequentially, and then have standalone sections that rely on knowledge of the core
+3. The other extreme, make the entire guides sequential
+
+We choose the middle ground, because it requires the least new writing, and if we chose to move towards one of the other extremes, it does not prevent that choice nor would we throw away work.
+
+It’s also useful to study the learning flow of other front end libraries in order to determine possible alternatives. Let’s look at a few.
+
+### React
+
+React is known for having low learning overhead for someone who is making their first app. With its popularity, we can guess that new users may expect to find similar topics easily accessible in our guides. This list is most useful for considering what should be in our Components section.
+
+1. Hello World
+2. Introducing JSX
+3. Rendering Elements
+4. Components and Props
+5. State and Lifecycle
+6. Handling Events
+7. Conditional Rendering
+8. Lists and Keys
+9. Forms
+    1. Lifting State Up
+    2. Composition vs Inheritance
+    3. Thinking In React
+
+The Overview section of the React Tutorial also helps show what we may be missing:
+
+- What Is React?
+- Inspecting the Starter Code
+- Passing Data Through Props
+- Making an Interactive Component
+- Developer Tools
+
+Nowhere on our current site do we have a highly visible explanation of what Ember is, beyond snippets. In light of this glaring omission, we have added a “What is Ember” section to the Guides Table of Contents above. It is not meant to replace the ongoing “Why Ember” and marketing-focused descriptions that are underway.
+
+### Vue
+
+As a fully-featured framework, Vue is an easier comparison for possible Table of Contents listings. Keep in mind that much of this type of content is present in our CLI docs instead, so this list will look longer than what we are aiming for.
+
+Introduction
```

Comment:
noticed these should use the same list syntax, PR here: https://github.com/jenweber/rfcs/pull/2/files